src/Post.vue

```diff
@@ -119,7 +125,7 @@ export default {
 }
 
 .inexor_article a {
```

Comment:
@movabo 'a' covers every states it can have (univisited = :link, :visited, :hover, :active) so it should not be a problem.
What is actually the problem is because of using <style> with the scoped attribute: 
>DOM content created with v-html are not affected by scoped styles, but you can still style them using deep selectors.
- https://vue-loader.vuejs.org/en/features/scoped-css.html

Reply:
Ah, indeed, I wasn't sure if browser-rules on a:visited override those rules, so my comment is actually totally wrong.